src/playground/services/invoices/update/endpoint.clj

```diff
@@ -17,4 +18,8 @@
                    (h/format))
         _      (jdbc/execute! db update)]
 
-    {:status 302 :headers {"Location" "/invoices"} :body "" :flash (str "Entry " id " has been updated.")}))
+
+    (-> (ring-resp/redirect "/invoices")
+        (assoc :flash (str "Entry " id " has been updated")))))
+
+#_{:status 302 :headers {"Location" "/invoices"} :body "" :flash (str "Entry " id " has been updated.")}
```

Comment:
Please never leave commented-out code when opening PRs. Actively prevent it by self-code-reviewing before opening a PR.